VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/participant-panel.model.spec.ts

```diff
@@ -38,6 +38,20 @@ describe('ParticipantPanelModel', () => {
         expect(model.isAvailable()).toBeTruthy();
     });
 
+    it('returns isAvailable: true when participant is quick link observer and in consultation', () => {
+        participant.status = ParticipantStatus.InConsultation;
+        participant.role = Role.QuickLinkObserver;
+        model = mapper.mapFromParticipantUserResponse(participant);
+        expect(model.isAvailable()).toBeTruthy();
+    });
+
+    it('returns isAvailable: true when participant is quick link observer and in consultation', () => {
+        participant.status = ParticipantStatus.InConsultation;
+        participant.role = Role.QuickLinkParticipant;
+        model = mapper.mapFromParticipantUserResponse(participant);
+        expect(model.isAvailable()).toBeTruthy();
+    });
```

Comment:
Add tests for when is a quicklink user but not in consultation and other roles when in consultation.

Although I would prefer to spyOnProperty isQuickLinkUser so these tests are focused on this specific logic and let the this.isQuickLinkUser tests cover the specific roles.